src/blocks/weather.rs

```diff
@@ -274,14 +272,16 @@ impl Weather {
                 let apparent_temp =
                     australian_apparent_temp(raw_temp, raw_humidity, raw_wind_speed, *units);
 
-                self.weather_keys = map!("{weather}" => raw_weather,
-                                  "{temp}" => format!("{:.0}", raw_temp),
-                                  "{humidity}" => format!("{:.0}", raw_humidity),
-                                  "{apparent}" => format!("{:.0}",apparent_temp),
-                                  "{wind}" => format!("{:.1}", raw_wind_speed),
-                                  "{wind_kmh}" => format!("{:.1}", kmh_wind_speed),
-                                  "{direction}" => convert_wind_direction(raw_wind_direction),
-                                  "{location}" => raw_location);
+                self.weather_keys = map!(
+                    "weather" => Value::from_string(raw_weather),
+                    "temp" => Value::from_integer(raw_temp as i64),
```

Comment:
Not .degrees() like the Temperature block?

Reply:
Forgot to add. And by the way, should it be `°C` instead of `°`?

Reply:
I've overridden it to °C in my config, but then again there are people who use fahrenheit so maybe just leave it the same as it is now?